### DISCUSSION TITLE
How to use nestjs on the server side?

### DISCUSSION THREAD

**Discussion by keykaykay** [nuxt/nuxt#22394] [Questions]
### Describe the feature

How to use nestjs on the server side

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by genu** [comment]
This sounds like a question for discord or a question in Discussions.

I'm not aware of any issues with using Nestjs with Nuxt.
